app/_src/policies/meshmetric.md

```diff
@@ -0,0 +1,284 @@
+
+The metrics endpoint is forwarded to the standard Envoy [Prometheus metrics endpoint](https://www.envoyproxy.io/docs/envoy/latest/operations/admin#get--stats?format=prometheus) and supports the same query parameters.
+You can pass the `filter` query parameter to limit the results to metrics whose names match a given regular expression.
+By default, all available metrics are returned.
+
+#### Secure metrics with TLS
+
+{{site.mesh_product_name}} allows configuring metrics endpoint with TLS.
+
+```yaml
+    backends:
+      - type: Prometheus
+        prometheus: 
+          port: 5670
+          path: /metrics
+          tls:
+            mode: ProvidedTLS
+```
+
+In addition to the `MeshMetric` configuration, `kuma-sidecar` requires a provided certificate and key for its operation.
+
+{% tabs expose-mesh-metrics-data-plane-proxies-tls useUrlFragment=false %}
+{% tab expose-mesh-metrics-data-plane-proxies-tls Kubernetes %}
+
+When the certificate and key are available within the container, `kuma-sidecar` needs the paths to provided files as the following environment variables:
+
+* KUMA_DATAPLANE_RUNTIME_METRICS_CERT_PATH
+* KUMA_DATAPLANE_RUNTIME_METRICS_KEY_PATH
+
+It's possible to use a [`ContainerPatch`](/docs/{{ page.version }}/production/dp-config/dpp-on-kubernetes/#custom-container-configuration) to add variables to `kuma-sidecar`:
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: ContainerPatch
+metadata:
+  name: container-patch-1
+  namespace: kuma-system
+spec:
+  sidecarPatch:
+    - op: add
+      path: /env/-
+      value: '{
+          "name": "KUMA_DATAPLANE_RUNTIME_METRICS_CERT_PATH",
+          "value": "/kuma/server.crt"
+        }'
+    - op: add
+      path: /env/-
+      value: '{
+          "name": "KUMA_DATAPLANE_RUNTIME_METRICS_KEY_PATH",
+          "value": "/kuma/server.key"
+        }'
+```
+
+{% endtab %}
+{% tab expose-mesh-metrics-data-plane-proxies-tls Universal %}
+
+Please upload the certificate and the key to the machine, and then define the following environment variables with the correct paths:
+
+	* KUMA_DATAPLANE_RUNTIME_METRICS_CERT_PATH
+	* KUMA_DATAPLANE_RUNTIME_METRICS_KEY_PATH
+
+{% endtab %}
+{% endtabs %}
+
+#### activeMTLSBackend
+
+We no longer support activeMTLSBackend, if you need to encrypt and authorize the metrics use [Secure metrics with TLS](#secure-metrics-with-tls) with a combination of [one of the authorization methods](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config).
+
+#### Running multiple prometheus instances
+
+If you need to run multiple instances of Prometheus and want to target different set of DPPs you can do this by using Client ID setting on both `MeshMetric` (`clientId`) and [Prometheus configuration](https://github.com/prometheus/prometheus/pull/13278/files#diff-17f1012e0c2fbd9bcd8dff3c23b18ff4b6676eef3beca6f8a3e72e6a36633334R2233) (`client_id`).
```

Comment:
I think we should use the full names whenever the abbreviation is not generally well known

```suggestion
If you need to run multiple instances of Prometheus and want to target different set of Data Plane Proxies you can do this by using Client ID setting on both `MeshMetric` (`clientId`) and [Prometheus configuration](https://github.com/prometheus/prometheus/pull/13278/files#diff-17f1012e0c2fbd9bcd8dff3c23b18ff4b6676eef3beca6f8a3e72e6a36633334R2233) (`client_id`).
```